packages/PreviewCard/PreviewCard.jsx

```diff
@@ -0,0 +1,160 @@
+  }
+  &:hover img {
+    transform: scale(1.03);
+  }
+`
+
+const P = styled.p`
+  color: ${colorTelusPurple};
+`
+
+/**
+ * The PreviewCard component creates the appearance of a page snippet, and can be used in a list format.
+ * @version ./package.json
+ */
+const PreviewCard = ({ category, other, image, body, footer, href, linkComponent, ...rest }) => {
+  if (rest.to && !(linkComponent && rest.to)) {
+    warn('Link', 'The props `linkComponent` and `to` must be used together.')
+  }
+
+  let newBody = body
+  if (body.length > 70) {
+    newBody = `${body.substr(0, 70)}...`
+  }
+
+  let header = category || other
+  if (category && other) {
+    header = `${category} \u00B7 ${other}`
+  }
+
+  const innerCard = (
+    <BoxContainer header={header} footer={footer}>
+      {image && (
+        <ImageContainer header={header} footer={footer}>
+          {image}
+        </ImageContainer>
+      )}
+      <Box horizontal={header || footer ? 4 : 3} vertical={header || footer ? 5 : 3}>
+        <ContentContainer header={header} footer={footer} data-testid="contentContainer">
+          {header && (
+            <Box>
+              <Text size="small" bold>
+                {header}
+              </Text>
+            </Box>
+          )}
+          <Box vertical={header || footer ? 3 : undefined}>
+            <P alt={body}>{newBody}</P>
+          </Box>
+          {footer && (
+            <Box>
+              <Text size="small">{footer}</Text>
+            </Box>
+          )}
+        </ContentContainer>
+      </Box>
+    </BoxContainer>
+  )
+
+  return React.createElement(
+    linkComponent || Anchor,
+    {
+      ...safeRest(rest),
+      href,
+    },
+    innerCard
+  )
+}
+
+PreviewCard.propTypes = {
+  /**
+   * Image component that will appear at the top of the card, above the content section.  Recommended dimensions is 369x269px.
+   */
+  image: PropTypes.node.isRequired,
+  /**
+   * Text that will appear at the top of the content section.  Recommended to be only one or two words.
+   */
+  category: PropTypes.string,
+  /**
+   *  Text that will appear at the top of the content section, next to category seperated by a dot. Recommended to be 3 or less words.
+   */
+  other: PropTypes.string,
+  /**
+   * Purple text that will appear in the middle of the content section.  Recommended amount of characters is 70 or less.
+   */
+  body: PropTypes.string.isRequired,
+  /**
+   * Text that will appear at the bottom of the content section.  Recommended to be 3 or less words.
+   */
+  footer: PropTypes.string,
+  /**
+   * Target URL.
+   */
+  href: PropTypes.string,
+  /**
+   * Link component.
+   */
+  linkComponent: PropTypes.func,
+  /**
+   * Target URL (if using 'Link from 'react-router').
+   */
+  to: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+}
+
+PreviewCard.defaultProps = {
+  category: undefined,
+  other: undefined,
+  footer: undefined,
+  href: undefined,
+  linkComponent: null,
```

Comment:
Is there are reason why `linkComponent` and `to` are _null_ whereas everything else is _undefined_?

Reply:
I originally used null in linkComponent and other props to be consistent with how links work in TDS core packages such as ButtonLink.  Having done a little bit of digging, it seems that we should be setting variables to null in default props.  I have made the change and will check in.
Source: http://ryanmorr.com/exploring-the-eternal-abyss-of-null-and-undefined/

Reply:
nice! thanks for sharing @Andrew-K-Lam 👍